app/helpers/argo_helper.rb

```diff
@@ -179,7 +179,7 @@ def render_foxml_link(document, link_text = 'FoXML', opts = {:target => '_blank'
   end
 
   def render_solr_link(document)
-    return '' unless current_user.is_admin
+    return '' unless can? :read, document
```

Comment:
The link to FoXML is a convenience to give access to any SDR admin without necessarily setting up direct Fedora admin access. So that criteria makes sense. I don’t know what or who the 'unless can? :read, document’  test adds. 

Lynn


> On Feb 26, 2016, at 11:39 AM, Darren L. Weber, Ph.D. <notifications@github.com> wrote:
> 
> In app/helpers/argo_helper.rb:
> 
> > @@ -179,7 +179,7 @@ def render_foxml_link(document, link_text = 'FoXML', opts = {:target => '_blank'
> >    end
> >  
> >    def render_solr_link(document)
> > -    return '' unless current_user.is_admin
> > +    return '' unless can? :read, document
> 
> Not clear whether this will behave the same way and is less clear about it's intent. It could be that this should not change.
> 
> —
> Reply to this email directly or view it on GitHub.
>